### ISSUE TITLE
Update docs for v1.7 software and v1.5 firmware

### ISSUE THREAD

**Issue by shawaj** [PiSupply/PiJuice#680]
Update docs in relation to #612 and #678

**Comment by shawaj** [comment]
[PiJuice Software V1.7 new features.pdf](https://github.com/PiSupply/PiJuice/files/6229376/PiJuice.Software.V1.7.new.features.pdf)

New features doc

**Comment by shawaj** [comment]
@ChristopherRush if you get a chance, could you look at adding the stuff in above PDF to the documentation?

I guess we also need to add details about the new battery profile and the ability to override the NTC resistance parameter too.

@mmilann @tvoverbeek anything else to add?

**Comment by mmilann** [comment]
Yes Custom profile for 12000mAh could be added to documentation, link: https://github.com/PiSupply/PiJuice/commit/d7940f95edb7ac28c419b4118b2607f582038cda
